src/plugins/intel_gna/src/gna_executable_network.hpp

```diff
@@ -128,8 +131,16 @@ class GNAExecutableNetwork : public InferenceEngine::IExecutableNetworkInternal
     }
 
     InferenceEngine::Parameter GetMetric(const std::string& name) const override {
-        if (ov::supported_properties == name) {
+        if (ov::supported_properties == name || InferenceEngine::Metrics::METRIC_SUPPORTED_CONFIG_KEYS == name) {
```

Comment:
We don't need add support of legacy API metrics.

Reply:
By legacy API, do you mean InferenceEngine::Metrics::METRIC_SUPPORTED_CONFIG_KEYS and InferenceEngine::Metrics::METRIC_SUPPORTED_METRICS?
So only ov::model_name is necessary?

Reply:
> By legacy API, do you mean InferenceEngine::Metrics::METRIC_SUPPORTED_CONFIG_KEYS and InferenceEngine::Metrics::METRIC_SUPPORTED_METRICS? So only ov::model_name is necessary?

Yes, I think it is quite strange introduce support of something what should be removed. I propose to add `ov::model_name` only.